android/gradle.properties

```diff
@@ -1,2 +1,3 @@
 android.useAndroidX=true
-android.enableJetifier=true
\ No newline at end of file
+android.enableJetifier=false
+android.jetifier.ignorelist=bcprov-jdk15on
```

Comment:
What's that reason of this change and what's its result?

Reply:
We don't need jetifier explicitely set enabled, but we DO need to ignore the bcprov-jdk15on as stated on the docs https://developer.onewelcome.com/android/android-sdk/q-a#failed-to-transform-bcprov-jdk15on

Reply:
Is it still a problem? According to the docs and linked Gradle issue (https://issuetracker.google.com/issues/159151549#comment12) this should be fixed in AGP 7+ (I can see that 7.2.2 is used currently in the project)